deployment/modules/gcp/cloudbuild/main.tf

```diff
@@ -21,7 +21,8 @@ module "artifactregistry" {
 # Cloud Build
 
 locals {
-  artifact_repo                = "${var.location}-docker.pkg.dev/${var.project_id}/${google_artifact_registry_repository.docker.name}"
+  cloudbuild_service_account   = "cloudbuild-${var.env}-sa@${var.project_id}.iam.gserviceaccount.com"
```

Comment:
Nit, maybe after this PR is merged - I would recommend passing value of entities created outside of this terraform via variables from Terragrunt directly. That way, it's easier for folks to re-use what we have with their own users. 

Even in Terragrunt, if you use some logic to build this name, people we'd need to use the same conventions for it to work. Hence, we need to explain what the naming convention is. Alternatively, you could just hardcode the full users, rather than building them programatically. We only have one environment for now, so that's not an issue.